src/css/components/complex_list.scss

```diff
@@ -76,3 +76,6 @@ $color-border: $color-textblack;
   }
 }
 
+.complex_list-item.saving {
+  background: #fad980;
```

Comment:
@rememberlenny Because this value (`#fad980`) is repeated at least one other place in the SCSS (line 90 of `src/css/components/loading.scss`), I suggest making it a variable.